sycl/include/CL/sycl/buffer.hpp

```diff
@@ -33,54 +33,58 @@ class buffer {
 
   buffer(const range<dimensions> &bufferRange,
          const property_list &propList = {})
-      : Range(bufferRange) {
+      : Range(bufferRange), MemRange(bufferRange) {
```

Comment:
What about writing modern C++ with some `{ }` instead of `()` for most of the instance constructions?